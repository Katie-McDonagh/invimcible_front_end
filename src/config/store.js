import { createStore, combineReducers } from 'redux'
import playerReducer from '../features/player/reducer'
import mapReducer from '../features/map/reducer'
import vimCommandReducer from '../components/vimCommands/reducer'

const rootReducer = combineReducers({
  player: playerReducer,
  map: mapReducer,
  vimCommand: vimCommandReducer
})

const store = createStore(
  rootReducer,
  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
)

export default store
