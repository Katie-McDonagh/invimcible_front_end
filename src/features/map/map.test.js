import React from "react";
import Map from "./index.js";
// import { connect } from "react-redux";
import { shallow, mount } from "enzyme";
import { Provider } from "react-redux";
// import { createStore, combineReducers } from "redux";
import store from "../../config/store";
import renderer from 'react-test-renderer';



var wrapper = null 

// beforeEach(() => {
//   wrapper = mount(<Provider store={store}>
//     <div>
//       <Map />
//     </div>
//   </Provider>)
// })

describe("Map", () => {
  it("renders without crashing", () => {
    shallow(
      <Provider store={store}>
        <div>
          <Map />
        </div>
      </Provider>
    );
  });

  xit("renders MapTile", () => {
    shallow( < MapTile />);
  });

  it('renders correctly', () => {
  });

  it("renders map information", () => {
    let component = renderer.create(
      <Provider store={store}>
        <Map />
      </Provider>
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
