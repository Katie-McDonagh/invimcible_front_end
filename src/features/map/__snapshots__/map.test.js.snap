// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map renders map information 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#42b842",
      "border": "4px solid white",
      "height": 384,
      "left": "-32px",
      "position": "relative",
      "top": "0px",
      "width": 736,
    }
  }
/>
`;
